src/main/java/cloud/fogbow/ras/constants/ConfigurationPropertyKeys.java

```diff
@@ -44,4 +44,5 @@
     public static final String AUTHORIZATION_ROLES_KEY = "roles";
     public static final String SUPERUSER_ROLE_KEY = "superuser_role";
     public static final String DEFAULT_AUTH_PLUGIN_KEY = "default_auth_plugin_class";
+    public static final String SUPERUSER_OPERATIONS_KEY = "superuser_operations";
```

Comment:
I believe it is safer to make these operations hardwired in the code, instead of configurable. Thus, I suggest to use a system constant to store the operations that require the user to be an admin to execute them.

Reply:
Done.